core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -424,6 +426,13 @@ case object KAFKA_2_7_IV2 extends DefaultApiVersion {
   val id: Int = 30
 }
 
+case object KAFKA_2_7_IV3 extends DefaultApiVersion {
```

Comment:
nit: this should be KAFKA_2_8_IV0, or KAFKA_2_8_IV1 if https://github.com/apache/kafka/pull/9601 merges